### PR TITLE
add RHEL 8.10 and 9.4 placeholders

### PR DESCRIPTION
**Pull request by croissanne**
These can be built by the ci image refresh bot.

---

The bot will open a PR like https://github.com/osbuild/gitlab-ci-terraform/pull/132

### CODE REVIEW

File: aws/rhel-8.10-ga-x86_64/.#config.json [hunk 1]
```diff
@@ -0,0 +1 @@
+sanne@worky.4070:1716798465
```
Comment (thozza) [inline aws/rhel-8.10-ga-x86_64/.#config.json:1]: Probably should not be part of the PR...

File: aws/rhel-8.10-ga-x86_64/#config.json# [hunk 2]
```diff
@@ -0,0 +1,6 @@
+o{
```
Comment (thozza) [inline aws/rhel-8.10-ga-x86_64/#config.json#:1]: Probably should not be part of the PR...

**Review by thozza — CHANGES_REQUESTED** [review]

File: aws/rhel-8.10-ga-x86_64/.#config.json (hunk 1 above)
Comment (croissanne) [reply aws/rhel-8.10-ga-x86_64/.#config.json:1]: woops ._.

File: aws/rhel-8.10-ga-x86_64/#config.json# (hunk 2 above)
Comment (croissanne) [reply aws/rhel-8.10-ga-x86_64/#config.json#:1]: I think I have my info/excludes set up badly :roll_eyes:

**Review by thozza — APPROVED** [review]
LGTM
